cmd/create/cluster/cmd.go

```diff
@@ -816,7 +816,7 @@ func run(cmd *cobra.Command, _ []string) {
 				}
 				selectedARN := ""
 				for _, rARN := range roleARNs {
-					if strings.Contains(rARN, fmt.Sprintf("%s-%s-Role", rolePrefix, role.Name)) {
+					if strings.Contains(strings.ToLower(rARN), strings.ToLower(fmt.Sprintf("%s-%s-Role", rolePrefix, role.Name))) {
```

Comment:
np: we could move the expected name outside the inner loop instead of rebuilding it for every arn checked. Also, should this be `strings.HasSuffix` instead of `Contains`?
And not directly related, but happens if the role name is truncated due to hitting the maximum length limit?

Reply:
For the acc roles in particular there is a validation to limit it to 32 characters so it would always have the suffix

Reply:
the complexity isn't too much of a consideration in this case, any implementation would work fast enough that it isn't noticeable. The comment was more regarding expressing the intent more clearly - iiuc, we expect the role arn to have a certain suffix, so we should use a method that expresses this expectation.

Reply:
We can also do this, what do you think?
https://github.com/openshift/rosa/pull/969/files#diff-943c0ab57b82c7028652df55e1d49e4f4e4dc41fa094dc4a9b3f2c35887d44b5R826

Reply:
even better!